src/open_inwoner/components/templates/components/Form/FileInput.html

```diff
@@ -17,7 +17,7 @@
     {% if field.errors %}{% errors errors=field.errors %}{% endif %}
 
     <div class="file-list" aria-live="polite">
-        <div class="file-list-selection" hidden><h4 class="file-list-selection__h4">{% if multiple %}{% trans 'Geselecteerde bestanden' %}{% else %}{% trans 'Geselecteerd bestand' %}{% endif %}</h4></div>
+        <div class="file-list-selection" hidden><h4 class="utrecht-heading-3 file-list-selection__heading">{% if multiple %}{% trans 'Geselecteerde bestanden' %}{% else %}{% trans 'Geselecteerd bestand' %}{% endif %}</h4></div>
```

Comment:
This changed a level 4 to a level 3

Reply:
Oh I noticed this matches the SCCS change later on.

Reply:
Yeah :-) this is a situation where the level of the tag stays the same, but it's just the styling that changes. So here the H4 needs to look like an H3. So I'm keeping this like this.